### DISCUSSION TITLE
Winmerge binary file - block definition

### DISCUSSION THREAD

**Discussion by smartino** [WinMerge/winmerge#1300] [Ideas]
I would like to use WinMerge to repair corrupted binary downloads.

It works but, as it defines a block ending at a byte match, it often breaks the difference in too many blocks to be coped manually.

Is there an option to set how many different bytes must match to define the end of a block?

or

Is it there a kind of macro, ie run the Alt -up, Alt-left  for 100 times?

TIA
Sergio Martino

**Comment by sdottaka** [comment]
Unfortunately, WinMerge does not have such a feature.

**Reply by sdottaka** [reply]
It is possible to select and copy multiple differences with Shift + cursor keys.
